spec/bundler/ruby_version_spec.rb

```diff
@@ -56,6 +56,40 @@
         end
       end
     end
+
+    describe ".from_string" do
+      shared_examples_for "returning" do
+        it "returns the original RubyVersion" do
```

Comment:
no it won't since the examples are included, instead of being nested in another context